cmd/integration_link.go

```diff
@@ -464,3 +523,20 @@ func validateHoursBeforeDelete(ans interface{}) error {
 	}
 	return nil
 }
+
+func isAutomaticReviewAppFromForksStillAllowedAwaringUserOfSecurityConcerns() (bool, error) {
+	io.Warning(warningSecurityMessageForAutomaticReviewAppsFromForks)
+	forksAllowed := false
+	err := survey.AskOne(&survey.Confirm{
+		Message: "Automatic review apps creation from forks:",
+		Default: forksAllowed,
+	}, &forksAllowed, nil)
+	if err != nil {
+		return false, err
+	}
```

Comment:
I like verbose and explicit function names, but that one is pretty long.
Also there are already some existing implementation for requesting confirmation to the user. It can be found in `region_migrations/confirmations.go` or `utils/consent.go` for example. I would suggest to take some inspiration from it:
```suggestion
func  askForConfirmation(message string) (bool, error) {
```


```suggestion
func  askForConfirmation(message string) (bool, error) {
	io.Warning(message)
	confirmed := false
	err := survey.AskOne(&survey.Confirm{
		Message: "Are your sure?",
		Default: forksAllowed,
	}, &forksAllowed, nil)
	if err != nil {
		return false, err
	}
```

Reply:
This is a nice idea, let's go for it